website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -135,6 +135,12 @@ metrics:
 
 </File>   
 
+:::caution
+
+- Metrics cannot be built on [ephemeral models](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/materializations#ephemeral). The materialization must have a represention in the data warehouse in order to build a metric.
```

Comment:
switch to active voice and fix representation typo

```suggestion
- You cannot build metrics on [ephemeral models](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/materializations#ephemeral). In order to build a metric, the materialization must have a representation in the data warehouse.
```

Reply:
Actually scratch that, I'm going to change built to define.